app/views/users/_user.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="user rounded row align-items-center m-5 mb-4 p-2 <%= myclass %>">
+  <%= image_tag(user.photo, alt: "user image", class: "col-3 user-images") %>
+  <div class="justify-content-start mb-5 col-6">
+    <h1><%= user.name %></h1>
+  </div>
+  <h6 class="align-items-end col-3">Number of posts: <%= user.posts_counter %></h6>
+</div>
```

Comment:
Nice job making use of a partial as requested by the previous reviewer😉. currently, the user page renders a blank page even after adding some users both on the console and making use of the seed😉. 

<img width="1437" alt="image" src="https://github.com/matee-safi/rails-blog/assets/64643995/171db1e1-2477-4ffc-9559-9d645e617988">

Kindly also note that this has made it impossible to see whether the `see all post` button works as requested by the previous reviewer😉. Please kindly do well to fix the issue👍🏼.

Reply:
@Ibizugbe I noted in my PR description that inserting data via rails console wasn't working, did you use pgadmin?

Reply:
Sorry about that, I didn't take note of that. 

Also, I don't use PG Admin. I just made use of the console and the seed.

Reply:
@Ibizugbe No problem. Because of the model validations it's not possible to insert the data as it says [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data). so I either have to manipulate the queries before inserting them, or comment out the validations for now, or insert data via pgadmin or psql cli with the required parameters. 
I chose the second option, I hope it doesn't cause any problems